composer.json

```diff
@@ -32,6 +32,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^4.8.35|^5.7|^6.5",
+        "sebastian/comparator": ">=1.2.3",
```

Comment:
Just to confirm this package should be under `require-dev` and not `require`?

Reply:
Yes, it's a dependency of phpunit (under sebastian's private github account)

Reply:
Okay, thanks.